topic_admin.go

```diff
@@ -0,0 +1,130 @@
+package kazoo
+
+import (
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/samuel/go-zookeeper/zk"
+)
+
+var (
+	ErrTopicExists          = errors.New("Topic already exists")
+	ErrTopicMarkedForDelete = errors.New("Topic is already marked for deletion")
+	ErrDeletionTimedOut     = errors.New("Timed out while waiting for a topic to be deleted")
+)
+
+// CreateTopic creates a new kafka topic with the specified parameters and properties
+func (kz *Kazoo) CreateTopic(name string, partitionCount int, replicationFactor int, topicConfig map[string]string) error {
```

Comment:
At the moment, probably not, as I just need to create and delete topics. However, there is a separate PR that contains some functionality to do so: https://github.com/wvanbergen/kazoo-go/pull/21 Perhaps you can open a PR and use #21 as a base. 😄